decidim-proposals/app/commands/decidim/proposals/admin/publish_participatory_text.rb

```diff
@@ -37,20 +38,28 @@ def call
         attr_reader :form
         attr_reader :publish_failures
 
-        def resort_proposals(form)
+        def update_contents_and_resort_proposals(form)
           form.proposals.each do |prop_form|
-            proposal = Decidim::Proposals::Proposal.where(component: current_component).find(prop_form.id)
+            proposal = Decidim::Proposals::Proposal.where(component: form.current_component).find(prop_form.id)
             proposal.set_list_position(prop_form.position) if proposal.position != prop_form.position
+            proposal.title = prop_form.title
+            proposal.body = prop_form.body if proposal.participatory_text_level == Decidim::Proposals::ParticipatoryTextSection::LEVELS[:article]
+
+            add_failure(proposal) unless proposal.save
+            raise ActiveRecord::Rollback if @publish_failures.any?
```

Comment:
Shouldn't this be *outside* of the loop? Otherwise it will raise at the first error

Reply:
Maybe a test case for this would be nice! 😄